src/network/http/websocket.cpp

```diff
@@ -197,25 +191,25 @@ namespace fc { namespace http {
       {
          public:
             websocket_server_impl()
-            :_server_thread( fc::thread::current() )
+               :_server_thread( fc::thread::current() ), fwd_header_key("")
             {
-
                _server.clear_access_channels( websocketpp::log::alevel::all );
                _server.init_asio(&fc::asio::default_io_service());
                _server.set_reuse_addr(true);
                _server.set_open_handler( [&]( connection_hdl hdl ){
-                    _server_thread.async( [&](){
-                       auto new_con = std::make_shared<websocket_connection_impl<websocket_server_type::connection_ptr>>( _server.get_con_from_hdl(hdl) );
-                       _on_connection( _connections[hdl] = new_con );
-                    }).wait();
+                  _server_thread.async( [&](){
```

Comment:
Please capture only what's required and wrap the long line.

Reply:
Original code passed [&], probably due to the _on_connection call. I could capture [this], but unsure it is much different than [&].

Reply:
It is.

Reply:
changed at https://github.com/bitshares/bitshares-fc/pull/134/commits/e336649b8f51f48a8cbd89467e9d82671c0db9e9